README.md

```diff
@@ -1,37 +1,62 @@
 # Exercism Elm Track
-[![build status](https://github.com/exercism/elm/workflows/elm%20%2F%20master/badge.svg)]
+
+![build status](https://github.com/exercism/elm/workflows/elm%20%2F%20master/badge.svg)
+
+![configlet](https://github.com/exercism/elm/workflows/configlet/badge.svg)
 
 Exercism Exercises in Elm
 
+This is the Elm track, one of the many tracks on [Exercism][web-exercism].
+
+It holds all the Elm _Concepts_, _Concept Exercises_ and _Practice Exercises_ that are currently implemented and available for students to complete.
+
+You can find the definitions of these in [`config.json`][config-json].
+
+This readme file is mainly targeted at people wishing to contribute, but feel free to take a look around if you're interested in how Exercism language tracks are set up.
+
+[web-exercism]: https://exercism.io
+
 ## Track Organization
 
 The track is organized with the following main directories and files:
+
```

Comment:
Some markdown linters require blank lines around code blocks, as the behaviour is different for different renderers. so have added some here